### PR TITLE
Update the private cluster instructions

### PR DESCRIPTION
**Pull request by jlewi**
* The cloud endpoints issue has been fixed so we can remove that bug.

Fix: #2086  Deploy Kubeflow in private GKE
Fix: #3081  Document how to deploy kubeflow with VPC service controls.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/682)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-682--competent-brattain-de2d6d.netlify.com/docs/gke/private-clusters/

File: content/docs/gke/private-clusters.md [hunk 1]
```diff
@@ -134,39 +134,34 @@ export PROJECT_NUMBER=$(gcloud projects describe kubeflow-dev --format='value(pr
 1. Create the access level
 
     ```
-    gcloud beta access-context-manager levels create cloudbuild \
+    gcloud beta access-context-manager levels create kubeflow \
        --basic-level-spec=members.yaml \
        --policy=${POLICYID} \
-       --title="Cloud Build"
+       --title="Kubflow ${PROJECT}"
```
Comment (sarahmaddox) [inline content/docs/gke/private-clusters.md]: Typo: "Kubeflow"

File: content/docs/gke/private-clusters.md [hunk 2]
```diff
@@ -134,39 +134,34 @@ export PROJECT_NUMBER=$(gcloud projects describe kubeflow-dev --format='value(pr
...
         ```
-        export ZONE_NAME=kubeflow
-        export NETWORK=<Network you are using for your cluster>
-        gcloud beta dns managed-zones create ${ZONE_NAME} \
-         --visibility=private \
-         --networks=https://www.googleapis.com/compute/v1/projects/${PROJECT}/global/networks/${NETWORK} \
-         --description="Kubeflow DNS" \
-         --dns-name=gcr.io \
-         --project=${PROJECT}
+          export ZONE_NAME=kubeflow
```
Comment (sarahmaddox) [inline content/docs/gke/private-clusters.md]: There seems to be too much space in front of these code lines in the rendered preview, now that the indentation is increased.

File: content/docs/gke/private-clusters.md [hunk 3]
```diff
@@ -231,13 +226,21 @@ export PROJECT_NUMBER=$(gcloud projects describe kubeflow-dev --format='value(pr
...
+
+
 1. Follow the [instructions](https://www.kubeflow.org/docs/gke/deploy/oauth-setup/) for creating an OAuth client
 
 1. Create environment variables for IAP OAuth access
 
     ```bash
-    export CLIENT_ID=<CLIENT_ID from OAuth page>
-    export CLIENT_SECRET=<CLIENT_SECRET from OAuth page>
+     export CLIENT_ID=<CLIENT_ID from OAuth page>
```
Comment (sarahmaddox) [inline content/docs/gke/private-clusters.md]: There seems to be too much space in front of these code lines in the rendered preview, now that the indentation is increased.

File: content/docs/gke/private-clusters.md [hunk 4]
```diff
@@ -50,17 +50,14 @@ Before installing Kubeflow ensure you have installed the following tools:
 You will need to know your gcloud organization id and project number; you can get it via gcloud
 
 ```
-export ORGANIZATION=$(gcloud organizations list --filter=DISPLAY_NAME=<ORG_NAME> --format='value(name)')
+export PROJECT=<your project id>
```
Comment (sarahmaddox) [inline content/docs/gke/private-clusters.md]:
```
s/<your project id>/<your GCP project ID>
```

**Comment by jlewi** [comment]
@sarahmaddox I think I fixed most of the indentation. except the members.yaml code block 

```
    - members:      
     - serviceAccount:${PROJECT_NUMBER}@cloudbuild.gserviceaccount.com
     - user:<your email>
```

The code block is outdented more then it should be.

**Comment by sarahmaddox** [comment]
@jlewi thanks! To fix the indent for the members code block, I think you need to indent just the lines with the three ticks by one more space. See the code block under "Create a managed private zone" for an example.

**Comment by jlewi** [comment]
@sarahmaddox that didn't seem to work; it didn't even end up being a code block. I tried a variety of things and couldn't fix it.

**Comment by sarahmaddox** [comment]
Oh dear, the joys of Markdown. :) Let's go with it as is for now, because it's legible and good info. Thanks for the hard work on this!

/lgtm
/approve

**Pull request merged**